<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowStartup</class>
 <widget class="QDialog" name="windowStartup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Feature Detector - Welcome</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/IFDstartup.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QCommandLinkButton" name="uiCommandLinkButtonOpen">
       <property name="text">
        <string>Open an image</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/fileopen.svg</normaloff>icons/fileopen.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="description">
        <string>Opens a stored image from the hard disk or memory card.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="uiCommandLinkButtonCaptureWebcam">
       <property name="text">
        <string>Capture an image from a camera</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/camera-video-symbolic.svg</normaloff>icons/camera-video-symbolic.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="description">
        <string>Shows the live image of the default camera and takes a picture to use it as a new image.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="uiCommandLinkButtonFastRT">
       <property name="text">
        <string>FAST features in real time</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/FastRT.png</normaloff>icons/FastRT.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="description">
        <string>Shows the live image of the default camera and overimposes in real time the FAST features calculated on every frame</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="uiToolButtonOpenRecent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Open Recent File</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/document-open-recent.svg</normaloff>icons/document-open-recent.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="uiCheckBoxStartup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Show this dialog on Startup</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uiCommandLinkButtonOpen</tabstop>
  <tabstop>uiCommandLinkButtonCaptureWebcam</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
